tv.kodi.Kodi.json

```
@@ -0,0 +1,193 @@
+                    "dest": "build/download/"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/xbmc/libdvdread/archive/6.0.0-Leia-Alpha-3.tar.gz",
+                    "sha256": "a30b6aa0aad0f2c505bc77948af2d5531a80b6e68112addb4c123fca24d5d3bf",
+                    "dest": "build/download/"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/xbmc/libdvdnav/archive/6.0.0-Leia-Alpha-3.tar.gz",
+                    "sha256": "071e414e61b795f2ff9015b21a85fc009dde967f27780d23092643916538a57a",
+                    "dest": "build/download/",
+                    "dest-filename": "libdvdnav-6.0.0-Leia-Alpha-3.tar.gz"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 tv.kodi.Kodi.appdata.xml /app/share/appdata/tv.kodi.Kodi.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "tv.kodi.Kodi.appdata.xml"
+                }
+            ]
+        },
+        {
+            "name": "kodiplatform",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/xbmc/kodi-platform",
+                    "commit": "e8574b883ffa2131f2eeb96ff3724d60b21130f7"
+                }
+            ]
+        },
+        "deps/fluidsynth.json",
+        "deps/libmodplug.json",
+        "deps/jsoncpp.json",
+        "deps/libopenmpt.json",
+        "addons/audiodecoder.asap.json",
+        "addons/audiodecoder.dumb.json",
+        "addons/audiodecoder.fluidsynth.json",
+        "addons/audiodecoder.gme.json",
+        "addons/audiodecoder.gsf.json",
+        "addons/audiodecoder.modplug.json",
+        "addons/audiodecoder.ncsf.json",
+        "addons/audiodecoder.nosefart.json",
+        "addons/audiodecoder.organya.json",
+        "addons/audiodecoder.qsf.json",
+        "addons/audiodecoder.snesapu.json",
+        "addons/audiodecoder.ssf.json",
+        "addons/audiodecoder.stsound.json",
+        "addons/audiodecoder.timidity.json",
+        "addons/audiodecoder.upse.json",
+        "addons/audiodecoder.vgmstream.json",
+        "addons/audiodecoder.wsr.json",
+        "addons/audioencoder.flac.json",
+        "addons/audioencoder.lame.json",
+        "addons/audioencoder.vorbis.json",
+        "addons/audioencoder.wav.json",
+        "addons/game.libretro.json",
+        "addons/inputstream.adaptive.json",
+        "addons/peripheral.joystick.json",
+        "addons/peripheral.xarcade.json",
+        "addons/pvr.argustv.json",
+        "addons/pvr.demo.json",
+        "addons/pvr.dvbviewer.json",
+        "addons/pvr.filmon.json",
+        "addons/pvr.hts.json",
```

Are there some problems with iptvsimple or you just forgot it?
```suggestion
        "addons/pvr.hts.json",
        "addons/pvr.iptvsimple.json",
```

---

It requires rapidxml, instead of the already present, tinyxml

---

and rapidxml is all headers only, so I can not make this a simple module without hacks. Ideally, this binary addon uses tinyxml instead, because kodi itself already uses it.